packages/client/wallets/aa/src/adapters/reservoir.ts

```diff
@@ -29,8 +29,7 @@ export function reservoirAdapter(wallet: EVMAAWallet): ReservoirWallet {
         },
         handleSendTransactionStep: async (chainId, stepItem, _) => {
             const stepData = stepItem.data;
-            return await wallet.sendTransaction({
-                chainId: chainId,
```

Comment:
have we tested this? We'll need to plug in to reservoir SDK

Reply:
No! To save time I'm waiting until the next release is ready, I'm keeping a tally of anything that needs manual testing and this is included. This'll also help inform plans for test automation

Reply:
🤠🤠🤠